icu4c/source/tools/icupkg/icupkg.cpp

```diff
@@ -198,6 +203,8 @@ static UOption options[]={
 
     UOPTION_DEF("matchmode", 'm', UOPT_REQUIRES_ARG),
 
+    UOPTION_DEF("ignore-deps", '\1', UOPT_NO_ARG),
```

Comment:
sorry, what does '\1' means here?

Reply:
I copied it from elsewhere. I think it means "no short form for this flag".